src/webgpu/api/operation/memory_sync/texture/texture_sync_test.ts

```diff
@@ -0,0 +1,144 @@
+export const kAccessPatterns = ['rw', 'wr'] as const;
+export type AccessPattern = typeof kAccessPatterns[number];
+
+// Boundary between the first operation, and the second operation.
+export const kOperationBoundaries = [
+  'queue', // Operations are performed on different queues.
```

Comment:
we can't really do this, so would need to go in a TODO somewhere.

Reply:
done